src/ocamlorg_frontend/components/secondary_footer.eml

```diff
@@ -1,30 +1,60 @@
 let render () =
-<footer class="bg-mild-contrast dark:bg-dark-mild-contrast border-t border-gray-200 pt-12 pb-20 md:p-0 md:fixed md:bottom-0 md:h-12 md:z-50 w-full" aria-labelledby="footer-heading">
-  <h2 id="footer-heading" class="sr-only">Footer</h2>
-  <div class="container-fluid wide">
-    <ul class="flex flex-col md:flex-row md:flex-wrap md:h-12 gap-x-6">
-      <li>
-        <a class="h-10 md:h-12 flex items-center mb-6 md:mb-0" href="<%s Url.index %>"><img class="h-8 md:h-6 pb-1" src="<%s Ocamlorg_static.Asset.url "logo-with-name.svg" %>" alt="OCaml"></a>
-      </li>
-      <li>
-        <a class="h-10 md:h-12 flex items-center text-lighter" href="<%s Url.getting_started %>" class="text-base leading-6 text-lighter">Get Started</a>
-      </li>
+  <footer class="flex flex-col gap-7 py-5 mt-6 border-solid border-t-[1px] border-[#00000033]">
+    <section class="flex justify-between items-end">
+      <div>
+        <a href="<%s Url.index %>"><img class="h-8" src="<%s Ocamlorg_static.Asset.url "logo-with-name.svg" %>" alt="OCaml"></a>
+        <p class="text-base leading-6 text-lighter mt-2">Innovation. Community. Security.</p>
+      </div>
+      <div class="flex items-center gap-4 pr-4">
+        <a href="https://github.com/ocaml/ocaml"> <%s! Icons.github "w-6 h-6" %></a>
+        <a href="https://discord.gg/cCYQbqN"> <%s! Icons.discord "w-6 h-6" %></a>
+        <a href="https://twitter.com/ocamllang"><%s! Icons.twitter "w-6 h-6" %></a>
+        <a href="/feed.xml"><%s! Icons.rss "w-6 h-6" %></a>
+      </div>
+    </section>
 
-      <li>
-        <a class="h-10 md:h-12 flex items-center text-lighter" href="<%s Url.api %>" class="text-base leading-6 text-lighter">Standard Library API</a>
-      </li>
+    <section class="flex justify-between w-full mt-6 md:w-[80%]">
+      <div class="flex flex-col gap-2">
```

Comment:
```suggestion
      <div class="flex flex-col gap-4">
```

Reply:
@sabine, I had reduced the padding and the gap because Claire mentioned it was too wide. Will revert the change.

Reply:
reducing the spacing / margin is almost always the better choice because it is not clickable. Padding on `a` element is clickable and helps people to more easily hit the target.